packages/react/react/src/utils/convertNewlineToJSX.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { create } from 'react-test-renderer';
+import { convertNewlineToJSX } from './convertNewlineToJSX';
+
+describe('convertNewlineToJSX', () => {
+  it('converts newline characters into <br /> JSX elements', () => {
+    const input = 'There was a problem trying to sign in.\nplease try again.';
+    const output = [
+      <React.Fragment key={0}>There was a problem trying to sign in.</React.Fragment>,
+      <React.Fragment key={1}>
+        <br />
+        please try again.
+      </React.Fragment>,
+    ];
+
+    const result = convertNewlineToJSX(input);
+
+    result.forEach((element, index) => {
```

Comment:
**I think the actual implementation of this function code is revealed in test code.**

I don't think it's good because the test is broken if the implementation changes. 
The purpose of this function is to add just `<br/>`, and `Fragment` doesn't seem to be interested in aspect of this function .

below paragraph is more natural I think!

```tsx
    const input = 'There was a problem trying to sign in.\nplease try again.';

    const result = convertNewlineToJSX(input);

    render(<>{result}</>);

    screen.debug();

    expect(screen.getByText('There was a problem trying to sign in.<br/>please try again.')).toBeInTheDocument();
```

Reply:
I didn't think about it Thank you so much!